github/compliance/report.py

```diff
@@ -116,14 +116,14 @@ def _template_vars(
     delivery_dashboard_url: str='',
 ):
     component = result_group.component
-    resource_name = result_group.resource_name
-    resources = [res.resource for res in result_group.results]
+    resource_name = result_group.name
```

Comment:
I'm confused, `<ScanResultGroup>.resource_name()` did return the `cm.Resource`.
Later in this PR, you replace it with `<ScanResultGroup>.artifact()` which returns `cm.Artifact`.

`<ScanResultGroup>.name` however contains `{component.name}:{resource.name}` which is not equal to `<ScanResultGroup>.resource_name()`.

Reply:
fixed in next commit